sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/models/confirmation-of-enrollment.dto.ts

```diff
@@ -33,6 +33,8 @@ export class ApplicationDetailsForCOEAPIOutDTO {
   applicationProgramDelivery: string;
   maxTuitionRemittanceAllowed: number;
   hasOverawardBalance: boolean;
+  disabilityProfileStatus: string;
```

Comment:
Shouldn't his be an Enum?

Reply:
I opted for having the enum and the descriptions in the API and just pass it as a string to the view.

Reply:
It was changed to use the enum.